patchkit-launcher-qt-src/src/lockfile.cpp

```diff
@@ -47,6 +47,17 @@ void LockFile::cede()
         {
             qCritical("Failed to cede the lock file.");
         }
+        cededFile.close();
         m_isLockFileLocal = false;
     }
 }
+
+void LockFile::clear()
```

Comment:
Same as before, remember to put note about this in change log since it's not related to pull request changes.